appeals-service-api/src/schemas/appeals.js

```diff
@@ -11,11 +11,11 @@ const AppealSchema = new Schema(
         unique: true,
       },
     },
-    appellantName: {
+    'appellant-name': {
       type: String,
       required: true,
     },
-    appellantEmail: {
+    'appellant-email': {
```

Comment:
by keeping the keys in the same format, we don't need to map between camel and kebab for no real reason